pontoon/base/models.py

```diff
@@ -1162,17 +1176,20 @@ def save(self, *args, **kwargs):
         for all project locales.
         """
         disabled_changed = False
+        is_dummy_changed = False
         if self.pk is not None:
             try:
                 original = Project.objects.get(pk=self.pk)
                 if self.disabled != original.disabled:
                     disabled_changed = True
+                if self.is_dummy != original.is_dummy:
```

Comment:
I don't think that's a use case we need to support. I don't see why anyone would change a project to `dummy` or the other way around. Do you expect that to happen?

Reply:
No, not at all. Will change it